3-projects/modules/standard_projects/data.tf

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+data "google_projects" "prod-project" {
+  filter = "labels.application_name:org-shared-vpc-prod"
+}
+
+data "google_project" "prod-project" {
```

Comment:
what is the purpose of this data lookup if the previous `google_projects` lookup gets the project ID?

Reply:
Great pick up!! You are right. I was using it to query other properties but no longer valid here